streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -74,19 +74,19 @@
      * will be automatically created in Kafka for failure recovery, where "applicationID"
      * is specified by the user in {@link org.apache.kafka.streams.StreamsConfig}.
      *
+     * @param <T>           the value type of the aggregated {@link KTable}
```

Comment:
Looks like IntelliJ reordered them automatically when I refactored the interfaces, do we have a standard intellij style template somewhere?